notebook/i18n/de/LC_MESSAGES/notebook.po

```diff
@@ -0,0 +1,416 @@
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2017-02-04 22:21-0600\n"
+"PO-Revision-Date: 2017-02-04 23:04-0600\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Globalization Pipeline\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
+"X-Generator: Poedit 1.8.11\n"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0, but you have < 1.1.0"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0 aber Sie < 1.1.0 haben"
+
+msgid "The Jupyter Notebook requires tornado >= 4.0, but you have %s"
+msgstr "Der Jupyter Notizblock erfordert Tornado >= 4.0 aber Sie %s haben"
+
+msgid "The `ignore_minified_js` flag is deprecated and no longer works."
+msgstr ""
+"Die `ignore_minified_js `Flagge wird missbilligt und funktioniert nicht mehr."
+
+msgid ""
+"Alternatively use `%s` when working on the notebook's Javascript and LESS"
+msgstr ""
+"Alternativ verwenden Sie `%s, `wenn auf dem Javascript des Notizblocks zu "
+"funktionieren und weniger"
+
+msgid ""
+"The `ignore_minified_js` flag is deprecated and will be removed in Notebook "
+msgstr ""
+"Die `ignore_minified_js `Flagge wird missbilligt und wird entfernt in "
+"Notizblock 6.0 sein"
+
+msgid ""
+"Widgets are unavailable. Please install widgetsnbextension or ipywidgets 4.0"
+msgstr ""
+"Dinger sind nicht verfügbar. Bitte installieren Sie widgetsnbextension oder "
+"ipywidgets 4.0"
+
+msgid "List currently running notebook servers."
+msgstr "Listen Sie auf und gegenwärtig führen Sie Notizblockserver aus."
+
+msgid "Produce machine-readable JSON output."
+msgstr "Stellen Sie maschinenlesbare JSON Ausgabe her."
+
+msgid ""
+"If True, each line of output will be a JSON object with the details from the "
+"server info file."
+msgstr ""
+"Wenn jede Zeile von Ausgabe wahr ist, wird sie ein JSON Objekt mit den "
+"Details von der Serverinformationsdatei sein."
+
+msgid "Currently running servers:"
+msgstr "Gegenwärtig Server ausführen:"
+
+msgid "Don't open the notebook in a browser after startup."
+msgstr "Öffnen Sie den Notizblock in einem Browser nicht nach Startup."
+
+msgid ""
+"DISABLED: use %pylab or %matplotlib in the notebook to enable matplotlib."
+msgstr ""
+"Unfähig gemacht: Verwenden Sie %pylab oder %matplotlib im Notizblock, um "
+"matplotlib zu ermöglichen."
+
+msgid "Allow the notebook to be run from root user."
+msgstr "Erlauben Sie, dass der Notizblock von Root geführt wird."
+
+msgid ""
+"The Jupyter HTML Notebook.\n"
+"    \n"
+"    This launches a Tornado based HTML Notebook Server that serves up an "
+"HTML5/Javascript Notebook client."
+msgstr ""
+"Die Jupyter HTML Notebook.n n dies startet einen tornadobasierten HTML "
```

Comment:
Moot since we're removing the machine translated German. This was an artifact of our old MT engine.